lib/internal/error-serdes.js

```diff
@@ -76,11 +76,11 @@ function GetName(object) {
   return desc && desc.value;
 }
 
-let util;
-function lazyUtil() {
-  if (!util)
-    util = require('util');
-  return util;
+let utilInspect;
+function lazyUtilInspect() {
+  if (!utilInspect)
+    utilInspect = require('internal/util/inspect');
+  return utilInspect;
 }
```

Comment:
As suggestion: this could be refactored to actually be a lazy loaded inspect function. Right now the name is a bit misleading.

Reply:
@BridgeAR You mean something like
```
let internalUtilInspect = null
function inspect (...args) {
  if (!internalUtilInspect) internalUtilInspect = require('inernal/util/inspect')
 internalUtilInspect.inspect(...args)
} 
```

Reply:
Yes :-)